tests/Command/MinifyAssetCommandTest.php

```diff
@@ -65,6 +65,21 @@ public function testMinifyAssetCommandFailsWhenInputFileDoesNotExist(): void
         $this->assertStringContainsString('Cannot read file', $tester->getDisplay());
     }
 
+    public function testMinifyAssetCommandFailsWhenInputFileIsNotCssOrJS(): void
+    {
+        $minifier = $this->createMock(MinifierInterface::class);
+
+        $command = new MinifyAssetCommand($minifier, __DIR__.'/../Fixtures');
+        $tester = new CommandTester($command);
+
+        $tester->execute(['input' => 'MinifyBundleTestKernel.php']);
+        $this->assertSame(Command::FAILURE, $tester->getStatusCode());
+        $display = $tester->getDisplay();
+        $this->assertStringContainsString('[ERROR] The type of "/', $display);
```

Comment:
The error message shows the full path, it depends on the environment where tests are run, I skipped it. Using a regex was too cumbersome.